subsys/net/ip/Kconfig.ipv4

```diff
@@ -14,6 +14,14 @@ if NET_IPV4
 config NET_INITIAL_TTL
 	int "Initial time to live for a connection"
 	default 64
+	range 0 255
+	help
+	  The value should be > 0
```

Comment:
Why allow the value to be set to `0` if the description states the opposite?

Reply:
0 is a valid value but not very useful usually so that is why it says "should". We actually should drop the packet if ttl is 0 (in TX), I just realized that we do not do that, I will send a patch for that. For RX, we drop the packet already if ttl of incoming packet is 0.

Reply:
I'm sorry, the description is confusing (it was before this change of course), and could use some additional context.

Reply:
Hmm, I could try to add more information there although I am not sure how much about IP protocol functionality should be described here. Anyway, I will figure out something.

Reply:
@pdgendt Would this description be helpful
```
	  The value should normally be > 0. The device receiving the IPv4
	  packet will decrement the value and will drop the packet if the TTL
	  value is 0. When sending, the packet is dropped before transmitting
	  to network if TTL is 0.
```

Reply:
LGTM